### PR TITLE
chore: remove clip_hg

### PR DESCRIPTION
**Pull request by ZiniuYu**
This PR removes the CLIP hugging face executor since hg does not have new models and every other model can be used from open-clip

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -42,7 +42,6 @@ jobs:
           python -m pip install wheel
           pip install --no-cache-dir "client/[test]"
           pip install --no-cache-dir "server/[onnx]"
-          pip install --no-cache-dir "server/[huggingface]"
```
Comment (numb3r3) [inline .github/workflows/cd.yml:45]: `mclip` would also need this dep

File: server/setup.py [hunk 2]
```diff
@@ -58,7 +59,6 @@
         ]
         + (['onnxruntime-gpu>=1.8.0'] if sys.platform != 'darwin' else []),
         'tensorrt': ['nvidia-tensorrt'],
-        'huggingface': ['transformers>=4.16.2'],
```
Comment (numb3r3) [inline server/setup.py:61]: change the dependency key to "mclip"

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: server/setup.py (hunk 2 above)
Comment (ZiniuYu) [reply server/setup.py:61]: why put mclip in extra requirements, since it is now part of basic models to support

File: server/setup.py [hunk 3]
```diff
@@ -58,7 +58,7 @@
...
+        'mclip': ['transformers>=4.16.2'],
```
Comment (numb3r3) [inline server/setup.py:61]:
```suggestion
        'transformers': ['transformers>=4.16.2'],
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -114,7 +114,7 @@ jobs:
           python -m pip install wheel pytest pytest-cov
           pip install --no-cache-dir "client/[test]"
           pip install --no-cache-dir "server/[onnx]"
-          pip install --no-cache-dir "server/[huggingface]"
+          pip install --no-cache-dir "server/[mclip]"
```
Comment (numb3r3) [inline .github/workflows/ci.yml:117]:
```suggestion
          pip install --no-cache-dir "server/[transformers]"
```

File: .github/workflows/cd.yml [hunk 5]
```diff
@@ -42,7 +42,7 @@ jobs:
...
+          pip install --no-cache-dir "server/[mclip]"
```
Comment (numb3r3) [inline .github/workflows/cd.yml:45]:
```suggestion
          pip install --no-cache-dir "server/[transformers]"
```

File: server/setup.py (hunk 2 above)
Comment (numb3r3) [reply server/setup.py:61]: I changed my mind, let's use `transformers` as the key. `mclip` actually depends on transformers.

**Review by numb3r3 — CHANGES_REQUESTED** [review]

**Review by numb3r3 — APPROVED** [review]
LGTM

**Pull request merged**